docs/advanced-usage/miscellaneous.md

```diff
@@ -17,6 +17,53 @@ void loop() {
 }
 ```
 
+# Organizing the sketch in good and safe order
+
+The example above is good to demonstrate usage of `Homie.isConnected()`. However, consider do not adding any code into `loop()` function. ESP8266 is relatively weak MCU, and Homie manages few important environmental tasks. Interferring with Homie might boomerang by sudden crashes. Advised sketch of Homie is:
```

Comment:
- consider to not add any code
- is a relatively weak MCU and Homie manages a few